notebooks/3_subset_data.ipynb

```diff
@@ -0,0 +1,1358 @@
+{
```

Comment:
<p>Maybe actually show this in code in two steps? (after this explanation)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Good point, I added this for the first example .</p>